Feature/add site content

amandujano opened the pull request:
new in this branch: 

* adding initial translate pages
* minor style changes

mxochicale commented on README.md:
```diff
@@ -1,12 +1,11 @@
 ## Getting started
-This is a Gatsby.js web development. In order to run this code you have to install the following: 
-
+This is a Gatsby.js web development. In order to run this code you have type the following commands in console:
 
```
Probably prerequisites section can be added such as the OS distro version in the machine

amandujano replied:
Check this [commit](https://github.com/air4children/site/pull/5/commits/4dca893d99a4acf8f99525366828e8c30a65ce32). I added a more descriptive information.

mxochicale commented on README.md:
```diff
@@ -20,9 +19,18 @@ This is a Gatsby.js web development. In order to run this code you have to insta
    `npm install -g gatsby-cli`
 1. Download this repository   
 1. Have a text editor installed, i.e. [VSCode](https://code.visualstudio.com/)
```
Maybe you can add 
Clone this repository
git clone https://github.com/air4children/site
Also, the recommended text editor VSCode but alternatively one can use favorite text editor (e.g , atom, pycharm, etc).

amandujano replied:
Yeah, thanks for this siggestion. I added [here](https://github.com/air4children/site/pull/5/commits/c988c4901982b6a22b7797df5424fcafa03b9a70) more information.

mxochicale commented on README.md:
```diff
@@ -20,9 +19,18 @@ This is a Gatsby.js web development. In order to run this code you have to insta
+
+Here's an image of the scheme color used in this website:
+
+![color scheme](.docs/color_code.png)
```
This looks great!

mxochicale commented on README.md:
```diff
@@ -1,12 +1,11 @@
 <h1 align="center">
-  Free Robotics web page
+  air4childrem web page
```
just a spelling error with children

amandujano replied:
Also fixed [here](https://github.com/air4children/site/pull/5/commits/4dca893d99a4acf8f99525366828e8c30a65ce32)

mxochicale commented on src/components/header.en.js:
```diff
@@ -0,0 +1,92 @@
+const Header = () => {
+  const { image } = useStaticQuery(graphql`
+    query {
+      image: file(relativePath: { eq: "air4childrenLogo1.png" }) {
```
Maybe you can update the logo with the latest version available here https://github.com/air4children/logos

amandujano replied:
oh nice. I updated the logo [here](https://github.com/air4children/site/pull/5/commits/8736298340f859e3d5b4310c3e047a795a6dc78e), but I am not using any images by now.

mxochicale replied:
Thanks Angel, for pointed out that. It is good news as logo will still change : )

mxochicale commented on src/pages/videos.mdx:
```diff
@@ -0,0 +1,23 @@
+# Videos 
```
Nice section, perhaps we do not need new lines.

amandujano commented:
> Hi @amandujano
> Such great job you have made. I just added few comments that are mainly cosmetic ones and the update of some logo of air4children. Perhaps, it is important to put a README for permission of other images that we do not own. By the way is "package-lock.json" a temporal file? Maybe we do not needed it as well and we can add a gitignore for that one and others.

Hey @mxochicale . I added some fixes with  your suggestions. You're right about the images, I am not using images by now, so I didnt use any reference for the permission. 

I was also thinking about the type of Licence the project would have. Do you have some guidance in this topic?

amandujano commented:
yep. I thinks is ready to be merged :)
